README.md

```diff
@@ -72,5 +73,5 @@ At a high level the services command currently:
 Important Notes:
 
 - We need to remove the local cache between requests. See https://github.com/rhd-gitops-example/services/issues/20. Until then, add `rm -rf ~/.promotion/cache; ` before subsequent requests.
-- New pull requests need new branches (i.e you cannot run the same command twice). Add `--branch [unique branch name]` before submitting further promotion PRs. See https://github.com/rhd-gitops-example/services/issues/21.
+- New pull requests need new branches (i.e you cannot run the same command twice). Add `--branch-name [unique branch name]` before submitting further promotion PRs. See https://github.com/rhd-gitops-example/services/issues/21.
```

Comment:
Again we should be able to remove --branch-name from this section of the readme since we create unique branches now.